config-template.json

```diff
@@ -0,0 +1,10 @@
+{
+	"hostName": "localhost",
+	"port": 8080,
```

Comment:
The reason for not naming it `config.json` is that you want to be able to have your own local version without risking to commit it. If the file is missing it will load default values so starting without a `config.json` is perfectly fine although it will generate a warning as it will result in authentication being turned off.